ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutorTest.java

```diff
@@ -279,47 +272,23 @@ public void shouldBuildQueriesWithPersistedConfig() {
     final ConfiguredStatement<?> configuredCsas = ConfiguredStatement.of(
         csasStatement, emptyMap(), expectedConfig);
 
-    expect(mockParser.parseSingleStatement(statementText)).andReturn(csasStatement);
+    when(mockParser.parseSingleStatement(statementText)).thenReturn(csasStatement);
+    when(mockEngine.getPersistentQueries()).thenReturn(ImmutableList.of());
     final KsqlPlan plan = Mockito.mock(KsqlPlan.class);
-    expect(mockEngine.plan(eq(serviceContext), eq(configuredCsas))).andReturn(plan);
-    expect(mockEngine.execute(eq(serviceContext), eqConfigured(plan)))
-        .andReturn(ExecuteResult.of(mockQueryMetadata));
-    mockQueryMetadata.start();
-    expectLastCall();
-
-    replay(mockParser, mockEngine, mockMetaStore, mockQueryMetadata);
+    when(mockEngine.plan(eq(serviceContext), eq(configuredCsas))).thenReturn(plan);
+    when(mockEngine.execute(eq(serviceContext), eqConfiguredPlan(plan)))
+        .thenReturn(ExecuteResult.of(mockQueryMetadata));
 
+    // When:
     handleStatement(statementExecutorWithMocks, csasCommand, csasCommandId, Optional.empty(), 1);
-
-    verify(mockParser, mockEngine, mockMetaStore, mockQueryMetadata);
-  }
-
-  private static class StatusMatcher implements IArgumentMatcher {
-    final CommandStatus.Status status;
-
-    StatusMatcher(final CommandStatus.Status status) {
-      this.status = status;
-    }
-
-    @Override
-    public boolean matches(final Object item) {
-      return item instanceof CommandStatus
-          && ((CommandStatus) item).getStatus().equals(status);
-    }
-
-    @Override
-    public void appendTo(final StringBuffer buffer) {
-      buffer.append("status(").append(status).append(")");
-    }
-  }
-
-  private static CommandStatus sameStatus(final CommandStatus.Status status) {
-    reportMatcher(new StatusMatcher(status));
-    return null;
+    
+    // Then:
+    verify(mockQueryMetadata, times(1)).start();
```

Comment:
strictly speaking, we're changing this test. It used to verify also that all of the other mocks happened exactly once (e.g. that `plan` was called, that `execute` was called and that `start` was called). I think this coverage is actually important since it helped me find a bug when I was doign #3933

Reply:
Discussed a bit, while it would be better to have explicit verification, Mockito will throw unnecessary stubbing exceptions if a mock isn't used so in a sense it "verifies" that the mock was called with the exact params. (Don't silence those exceptions)